tools/tcpconnect_example.txt

```diff
@@ -94,6 +111,8 @@ optional arguments:
   -c, --count           count connects per src ip and dest ip/port
   --cgroupmap CGROUPMAP
                         trace cgroups in this BPF map only
+  --mntnsmap MNTNSMAP   trace mount namespaces in this BPF map only
+  -d, --dns             include likely DNS query associated with each connect
```

Comment:
please update the man page as well: man/man8/tcpconnect.8

Reply:
done